recipes/fastlinkingnumbers/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "fastlinkingnumbers" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/antequ/fastlinkingnumbers/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: b3393bffb8669ab8214b1b86ae8b1d48abd70ba021134a3f2a5e5fcfd641e7d9
+  patches:
+    - CMakeLists.txt.patch
+    - test.sh.patch
+    - llvm.patch
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make
+    - cmake
+  host:
+    - llvm-openmp >=3.7  # [osx]
+    - libgomp >=4.7      # [linux]
+    - gflags
+    - eigen >=3
+
+test:
+  source_files:
+    - data
+    - results
+    - test.sh
+  commands:
+    - sh test.sh
```

Comment:
Patch / choose tests so it actually returns an error state if there is an error. For example, the test output for the latest build is:

```
+ sh test.sh
Glove: (Mean error 0.00214443)
Certificate matches the reference for glove.
Knit Tube Init: (Mean error 0.00366683)
Certificate matches the reference for knittubeinit.
Knit Tube Broken: (Mean error 0.006343)
test.sh: line 26: ./verifycurves: No such file or directory  # this line is an error?
Certificate does not match the reference for knittubebroken.
Simplified Woundball (blenderobj): (Mean error 0.000668562)
Certificate matches the reference for blenderobj.
Simplified Woundball (houdiniobj): (Mean error 0.00138268)
Certificate matches the reference for houdiniobj.
+ exit 0
```
One of the tests didn't run, but the error state of the script was zero, so the test passed.